### DISCUSSION TITLE
TypeDoesNotContainType - how to solve this?

### DISCUSSION THREAD

**Discussion by michabbb** [vimeo/psalm#8156] [Q&A]
hi there,

i would be happy if someone could let me know, how I can solve this:
https://psalm.dev/r/0547b7734b

thanks a lot !!

**Comment by AndrolGenhald** [comment]
Well firstly I think the class you want to check for is probably `\this\is\my\test\bla` rather than `	his\is\my	est\bla`, so either use single quotes or escape the backslashes.

The real problem though is that this is a bug in Psalm.
